server/controllers/userController.js

```diff
@@ -82,6 +82,31 @@ class UserController {
       signupMethod: user.signupMethod
     });
   }
+
+  static async emailSignin(req, res) {
+    try {
+      const { email, password } = req.body;
+      const user = await models.Users.findOne({ where: { email } });
+      if (!user) return util.errorstatus(res, 401, 'Incorrect Login information');
```

Comment:
this was changed in my recent pr to follow the standard camelCase naming, so `util.errorstatus` is now `util.errorStatus`.

Reply:
@ocranbillions Pls effect the change let's move on.